### PR TITLE
Add currency symbols and alphabetise list

### PR DESCRIPTION
**Pull request by ninagittle**

### CODE REVIEW

File: map.js [hunk 1]
```diff
@@ -1,121 +1,165 @@
 module.exports =
-{ "ALL": "L"
+{ "AED": "د.إ"
 , "AFN": "؋"
+, "ALL": "L"
+, "AMD": "֏"
```
Comment (bengourley) [inline map.js]: This character doesn't appear to work, just displays at a square for me.

File: map.js [hunk 2]
```diff
@@ -1,121 +1,165 @@
...
+, "CDF": "FC"
+, "CHF": "Fr."
 , "CLP": "$"
 , "CNY": "¥"
 , "COP": "$"
 , "CRC": "₡"
-, "HRK": "kn"
+, "CUC": "$"
 , "CUP": "₱"
+, "CVE", "$"
```
Comment (bengourley) [inline map.js]: There's a syntax error on this line, the comma in the middle should be a colon.

**Comment by ninagittle** [comment]
Hi I made the changes you requested. And thanks for congratulating me on my first pr :grin:

**Comment by bengourley** [comment]
Perfect thanks!

**Pull request merged**